src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,20 @@
+package Duke;
+
+import java.util.Scanner;
+import Duke.Tasks.*;
+import Duke.Exceptions.*;
+
+public class Duke {
+    public static void main(String[] args) {
+        Ui.printWelcome();
+        Scanner sc = new Scanner(System.in);
+        Ui iu = new Ui();
+        Parser her = new Parser();
+        TaskList tasks = new TaskList();
+        boolean running = true;
```

Comment:
Will using "isRunning" rather than "running" make the name sound more boolean here?